### ISSUE TITLE
Argument of type '{ modules: string[]; ........ }' is not assignable to parameter of type 'NuxtConfig'.   Object literal may only specify known properties, and 'modules' does not exist in type 'NuxtConfig'

### ISSUE THREAD

**Issue by narasimhajupally** [harlan-zw/nuxt-seo-kit#8]
### Describe the bug

As i put the nuxt-seo-kit config in extends option . Typescript linter is showing error.

Argument of type '{ modules: string[]; ........my ..... config.... }' is not assignable to parameter of type 'NuxtConfig'.   Object literal may only specify known properties, and 'modules' does not exist in type 'NuxtConfig'

I have run `yarn postinstall` to generate types but still the error is there.

Is NuxtConfig type is changed by seo-kit ?

Thanks

### System Info

```Shell
System:
    OS: Linux 6.0 Manjaro Linux
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i5-11300H @ 3.10GHz
    Memory: 4.96 GB / 15.41 GB
    Container: Yes
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 16.14.2 - ~/.nvm/versions/node/v16.14.2/bin/node
    Yarn: 1.22.19 - ~/.yarn/bin/yarn
    npm: 8.7.0 - ~/.nvm/versions/node/v16.14.2/bin/npm
  Browsers:
    Firefox: 108.0.1
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/harlan-zw/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [ ] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by modbender** [comment]
I had the same problem, I had to delete `tsconfig.json` file I had in my root and then add this to the start of `nuxt.config.ts`

```ts
import { defineNuxtConfig } from "nuxt/config";
```

**Comment by harlan-zw** [comment]
Hey, thanks for raising this issue.

Should be fixed in 0.4.11

**Comment by narasimhajupally** [comment]
I have noticed this in the config.
```
siteUrl: process.env.NUXT_PUBLIC_SITE_URL || 'http://localhost:3000/',
siteDescription: process.env.NUXT_PUBLIC_SITE_URL || 'My Nuxt v3 website.',
```

same env variable for URL and description

**Comment by harlan-zw** [comment]
Thanks @narasimhajupally 

Have pushed up a fix for that in 0.4.12
